packages/core/lib/command-utils.js

```diff
@@ -201,6 +203,37 @@ const runCommand = async function (command, options) {
   return await command.run(options);
 };
 
+const processHelpInput = inputStrings => {
+  //When we think user is trying to access help incorrectly, provide them with some helpful information.
```

Comment:
can you also comment on which exact case each of these `if`s is trying to handle? I can't keep straight which is trying to handle which. the comments on the other two are the same, one of them seems like it is probably incorrect as they aren't doing the same thing